src/Idris/REPL.hs

```diff
@@ -942,19 +942,21 @@ process fn (Core t)
 
 process fn (DocStr (Left n) w)
    = do ist <- getIState
-        let docs = lookupCtxtName n (idris_docstrings ist) ++
-                   map (\(n,d)-> (n, (d, [])))
-                       (lookupCtxtName (modDocN n) (idris_moduledocs ist))
-        case docs of
-          [] -> iPrintError $ "No documentation for " ++ show n
-          ns -> do toShow <- mapM (showDoc ist) ns
-                   iRenderResult (vsep toShow)
-    where showDoc ist (n, d) = do doc <- getDocs n w
-                                  return $ pprintDocs ist doc
-
-          modDocN (NS (UN n) ns) = NS modDocName (n:ns)
-          modDocN (UN n)         = NS modDocName [n]
-          modDocN _              = sMN 1 "NotFoundForSure"
+        case show n of
```

Comment:
Shouldn't this pattern-match the name itself and check for the specific name that you created, rather than relying on the Show instance not changing? That would be less prone to breakage if we e.g. add universe level display.